variables.tf

```diff
@@ -29,6 +29,12 @@ variable "ingress_cidr_blocks" {
   default     = []
 }
 
+variable "ingress_self" {
+  type        = bool
+  description = "Specify whether the security group itself will be added as a source to the ingress rule."
+  default     = true
```

Comment:
Can you please set this to `false` as a default

Reply:
Fixed. Thanks.